app/com/lucidchart/open/cashylive/controllers/AssetController.scala

```diff
@@ -96,10 +100,46 @@ class AssetController extends AppController {
   }
 
   /**
+   * Make a request for a resource and retry until either a successful response is given or the
+   * limit is reached. Pass all headers through both ways.
+   *
+   * @param url the url of the resource to request.
+   * @param request the original request made for the resource
+   * @param retriesRemaining the number of retries remaining
+   * @return the response from cloudfront
+   */
+  private def proxyRequestWithRetry(
+      url: String,
+      request: Request[_],
+      retriesRemaining: Int)(
+      implicit ec: ExecutionContext): Future[(WSResponseHeaders, Enumerator[Array[Byte]])] = {
+    if (retriesRemaining == 1) {
```

Comment:
retriesRemaining <= 0

Consider the case where retries is configured to 0 or 1